tests/foreman/api/test_repository_set.py

```diff
@@ -86,9 +90,13 @@ def test_positive_reposet_disable(self):
             name=REPOSET['rhva6'],
             product=product,
         ).search()[0]
-        reposet.enable(data={'basearch': 'x86_64', 'releasever': '6Server'})
-        reposet.disable(data={'basearch': 'x86_64', 'releasever': '6Server'})
-        repositories = reposet.available_repositories()['results']
+        data = {'basearch': 'x86_64',
```

Comment:
as the product id does not change, why we are recreating data var, if you are not sure that it can be modified by the called functions you can pass a copy of it.
```python
reposet.enable(data=data.copy())
repositories = reposet.available_repositories(data=data.copy())['results']
```

Reply:
I'm not so sure product id will match across different orgs and different uploaded manifests. Maybe @ldjebran meant smth else or didn't noticed that data var is not recreated, but  those are 2 different vars from different tests.
Anyway, that's not a blocking comment but rather a small suggestion. As Djebran is currently on PTO and we need this fix ASAP as it affecting lots of tests - let's skip it in scope of this PR and address later if needed.